### ISSUE TITLE
Missing pretrained weight file for reproducing CADETS_E5 experiment

### ISSUE THREAD

**Issue by 1zzan** [ubc-provenance/orthrus#20]
Hi Orthrus team,

Thank you for releasing the code and artifacts for the Orthrus paper. We are currently trying to reproduce the experiments reported in the README, especially the E5 experiments.

In the README, the `CADETS_E5` experiment is listed under the section using pre-trained Orthrus weights:

```bash
PYTHONHASHSEED=0 python src/orthrus.py CADETS_E5 --from_weights --detection.gnn_training.node_out_dim=128 --detection.gnn_training.lr=0.0001 --detection.gnn_training.encoder.graph_attention.dropout=0.1 --graph_construction.build_graphs.time_window_size=1.0
However, we could not find the corresponding pretrained weight file in the repository. The weights/ directory currently contains:

CADETS_E3.pkl
CLEARSCOPE_E3.pkl
CLEARSCOPE_E5.pkl
THEIA_E3.pkl
THEIA_E5.pkl
but there does not seem to be a CADETS_E5.pkl file. Since the code appears to load weights from:
weights/{cfg.dataset.name}.pkl

running CADETS_E5 --from_weights seems to require weights/CADETS_E5.pkl.

Could you please confirm whether the pretrained CADETS_E5.pkl file is intended to be released? If so, would it be possible to provide it, or point us to where it can be downloaded?

Also, if reproducing the E5 experiments requires any additional preprocessed .pkl files or artifacts beyond the raw dataset downloads and the files in weights/, we would appreciate any guidance.

Thanks again for your work and for making the project available.

**Comment by TristanBilot** [comment]
Hi @1zzan,

Unfortunately, we don't have access to the original CADETS_E5 pkl file anymore. 
The only way to reproduce similar results on this dataset is to perform hyperparameter tuning across gnn embedding dimensions. 

Sorry for the inconvenience!
